pdr_backend/pdr_dashboard/pages/predictoors.py

```diff
@@ -132,108 +128,46 @@ def get_feeds_table_area(
-    ) -> Dict[str, Union[float, int, str]]:
-        # TODO: remove/adjust
-        result = find_with_key_value(feed_subcription_stats, "contract", contract)
-
-        if result:
-            sales_str = f"{result['sales']}"
-
-            df_buy_count_str = (
-                f"{result['df_buy_count']}-DF" if result["df_buy_count"] > 0 else ""
-            )
-            ws_buy_count_str = (
-                f"{result['ws_buy_count']}-WS" if result["ws_buy_count"] > 0 else ""
-            )
-
-            if df_buy_count_str or ws_buy_count_str:
-                counts_str = "_".join(
-                    filter(None, [df_buy_count_str, ws_buy_count_str])
-                )
-
-                if counts_str:
-                    sales_str += f"_{counts_str}"
-
-            return {
-                "price_(OCEAN)": result["price"],
-                "sales": sales_str,
-                "sales_raw": result["sales"],
-                "sales_revenue_(OCEAN)": result["sales_revenue"],
-            }
-
-        return {
-            "price_(OCEAN)": 0,
-            "sales": 0,
-            "sales_raw": 0,
-            "sales_revenue_(OCEAN)": 0,
-        }
-
-    def get_feeds_data_for_feeds_table(
+    def get_data_for_predictoors_table(
         self,
-        feed_stats: List[Dict[str, Any]],
-        feed_subcription_stats: List[Dict[str, Any]],
+        predictoor_stats: List[Dict[str, Any]],
     ) -> Tuple[List[Dict[str, str]], List[Dict[str, Any]], List[Dict[str, Any]]]:
         # TODO: remove/adjust
 
-        temp_data = self.app.feeds_data
+        temp_data = predictoor_stats
 
-        new_feed_data = []
-        ## split the pair column into two columns
-        for feed in temp_data:
-            split_pair = feed["pair"].split("/")
-            feed_item = {}
-            feed_item["addr"] = feed["contract"]
-            feed_item["base_token"] = split_pair[0]
-            feed_item["quote_token"] = split_pair[1]
-            feed_item["exchange"] = feed["source"].capitalize()
-            feed_item["time"] = feed["timeframe"]
-            feed_item["full_addr"] = feed["contract"]
-
-            result = self.get_feeds_stat_with_contract(feed["contract"], feed_stats)
+        new_predictoor_data = []
 
-            feed_item.update(result)
+        appr_cost = calculate_tx_gas_fee_cost_in_OCEAN(self.app.web3_pp, "0x18f54cc21b7a2fdd011bea06bba7801b280e3151", self.app.prices)
 
-            subscription_result = self.get_feeds_subscription_stat_with_contract(
-                feed["contract"], feed_subcription_stats
-            )
+        ## split the pair column into two columns
+        for data_item in temp_data:
+            print("data_item--->", data_item)
 
-            feed_item.update(subscription_result)
+            temp_pred_item = {}
+            temp_pred_item["addr"] = str(data_item["user"])
+            temp_pred_item["gross_income_(OCEAN)"] = data_item["total_payout"]
```

Comment:
should we remove the (OCEAN) part?

Reply:
According to the task, there should be the `(OCEAN)` phrase on the column header

https://github.com/oceanprotocol/pdr-backend/issues/1533

Reply:
on the column header, yes... but is it really necessary as a data key? With filtering and formatting and all that, I think adding parantheses (which are special characters) is maybe not such a good idea

Reply:
I'm fine either way, just asking for your opinion too

Reply:
I can agree that it is not pretty 👍 we used like that on the feeds table too